doc/source/cookbook.rst

```diff
@@ -488,13 +531,13 @@ Unlike agg, apply's callable is passed a sub-DataFrame which gives you access to
 
 .. ipython:: python
 
-   S = pd.Series([i / 100.0 for i in range(1,11)])
+   S = pd.Series([i / 100.0 for i in range(1, 11)])
 
-   def CumRet(x,y):
-      return x * (1 + y)
+   def CumRet(x, y):
+       return x * (1 + y)
 
    def Red(x):
```

Comment:
use `red`, I don't think there is a reason to have these functions named as classes